jobs/infra/fixtures/cleanup-aws.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+if [[ $0 == $BASH_SOURCE ]]; then
+  echo "$0 should be sourced";
+  exit
+fi
+echo "sourced ${BASH_SOURCE:-$0}"
+
+
+function purge::aws
+{
+    default_region=us-east-1
```

Comment:
as discussed in Engineering sprint, we should ensure we purge all possible aws regions